about.html

```diff
@@ -0,0 +1,121 @@
+		<h1> RAY'S ELEMENT AQUA <br /> GLOBAL SUMMIT 2021 </h1>
+	</div>
+	<p class="desktop-article">Element Aqua Summit is the most highly influential and profitable network for
+		Carribean's health,fitness and wellness industry.
+		we organize health, fitness and wellness conferences, events that moves even the most fable minds to
+		achieve their highest potential as far as living healthy is concerned. Our Awards duely recognise
+		leading companies and excutives from accross the health, fitness and wellness ecosystem.
+	</p>
+	<div class="article2">
+		<p>Please reach us through email for any futher questions and informations you need</p>
+		<a class="reach-out" href="mailto">rayselementsummit2021@aqua.org</a>
+	</div>
+	<section class="logo-wrapper">
+		<div class="aqua-logo">
+			<p id="logo-p">The Ray's Element Aqua Summit 2021 logo</p>
+		</div>
+		<div class="text">
+			<p class="mobile-text"> The logo of Ray's Aqua summit 2021 was decided through a series of logo competition
+				by stake holders from 8 may to 12 june.</p>
+		</div>
+		<p id="text-logo"> The logo of Ray's Aqua summit 2021 was decided through a series of logo competition
+			by stake holders from 8 may to 12 june. After online public voting and selection commitee's judging
+			process, the logo from photographer Vincent won the price. After consultation with Mr john we have
+			optimized the logo for this year's summit as follow</p>
+		<div class="mobile-image">
+			<img class="logo-pic" src="Assets/Icons/running.jpeg" alt="summit-logo">
+		</div>
+	</section>
+	<hr id="line">
+	<div class="past-summits">
+		<h3>See the past RAY'S Aqua Global Summits</h3>
+		<p>Take a look at the last two summits which took place in Dominincan repulblic and Barbados</p>
+	</div>
+	<section class="Summit-examples">
+		<div class="images">
+			<img src="Assets/Images/barbardos.jpeg" alt="barbados-summit">
+			<img id="d-down" src="Assets/Images/dominican.jpeg" alt="carribean-summit">
+		</div>
+	</section>
+	<section class="partner">
+		<div class="footer-mobile">
+			<h2>Partners</h2>
+			<ul class="partners-mobile">
+				<li><a href="https://www.mozilla.org/en-US/firefox/new/" id="dark">mozilla</a></li>
+				<li><a href="https://www.google.com/">Google</a></li>
+				<li><a href="https://www.naver.com/" id="lightgrey">NAVER</a></li>
+				<li><a href="https://www.crunchbase.com/organization/daum-kakao">daumkakao</a></li>
+				<li><a href="https://www.airbnb.com/">Airbnb</a></li>
+			</ul>
+		</div>
+	</section>
+	<footer class="footer-container">
+		<div class="footer-child">
+			<h3><span class="circle">cc</span>Ray's<span>aqua</span> </h3>
+		</div>
+		<div class="footer-child">
+			<p class="rights">2020 elements aqua creations. all rights reserved</p>
+			<p class="message">Every content in this page is the sole property of ray's aqua. transfer,use, publication
+				without the
+				credits is hereby prohibited and violations is subject to charges under law</p>
+		</div>
+	</footer>
```

Comment:
- [ ] _[Optional]_ Both pages should contain the same navbar and `footer` but this part (footer) of your `about.html` page is not present on the `index.html` page (on mobile view). Kindly add it.

Reply:
> * [x]  _[Optional]_ Both pages should contain the same navbar and `footer` but this part (footer) of your `about.html` page is not present on the `index.html` page (on mobile view). Kindly add it.

Follow the template, you all keep screaming to cover your incompetence, if you follow the template was this feature implemented in it?  I'm not even cut out for any of this BS anymore.